statisticsapp/src/main/java/com/example/statisticsapp/domain/usecase/GetTasksForPlantUseCase.kt

```diff
@@ -0,0 +1,8 @@
+package com.example.statisticsapp.domain.usecase
+
+import com.example.plantsapp.domain.model.Plant
+import com.example.plantsapp.domain.model.Task
+
+interface GetTasksForPlantUseCase {
```

Comment:
Just for your info - maybe you can move some use-cases interfaces to domain module, to reuse them in both applications